rockcraft/cli.py

```diff
@@ -57,6 +58,20 @@
 ]
 
 
+def _emit_error(error: craft_cli.CraftError, cause: Optional[Exception] = None) -> None:
+    """Emit the error in a centralized way so we can alter it consistently."""
+    # set the cause, if any
+    if cause is not None:
+        error.__cause__ = cause
+
+    # if running inside a managed instance, do not report the internal logpath
+    if is_managed_mode():
+        error.logpath_report = False
```

Comment:
Can we just do :

error.logpath_report = (not is_managed_mode()) ?

Reply:
In the hypothetical case where the variable already comes disabled for whatever reason, we may not want to turn it back on.